### ISSUE TITLE
Crashed on Microsoft Store update

### ISSUE THREAD

**Issue by Hexaae** [t1m0thyj/WinDynamicDesktop#512]
- WDD Version: 5.2.1
- OS Version: Win 11 22H2

Steps to Reproduce:

1. just updated the app to latest version from the MS Store: it silently (you'll notice rolling circle pointer but see no crash requester. You'll find the crash log in Event Viewer or Reliability Monitor) crashed during update (I was running WDD 5.2.0 in background). Installation was successful though.
```Descrizione
A problem caused this program to stop interacting with Windows.
Percorso dell'applicazione che ha generato l'errore:	C:\Program Files\WindowsApps\38719TimothyJohnson.WinDynamicDesktop_5.2.0.0_x64__rfarkse4rn21a\WinDynamicDesktop\WinDynamicDesktop.exe

Firma del problema
Nome evento problema:	MoAppHang
Package Full Name:	38719TimothyJohnson.WinDynamicDesktop_5.2.0.0_x64__rfarkse4rn21a
Application Name:	praid:App
Application Version:	5.2.0.0
Application Timestamp:	6377dc84
Hang Signature:	196a
Hang Type:	2097152
Versione SO:	10.0.22621.2.0.0.256.48
ID impostazioni locali:	1040
Additional Hang Signature 1:	196a76c64739cd1c43af4ec94a4e963a
Additional Hang Signature 2:	4b67
Additional Hang Signature 3:	4b67857320e07e877cd0641d73a6b29a
Additional Hang Signature 4:	196a
Additional Hang Signature 5:	196a76c64739cd1c43af4ec94a4e963a
Additional Hang Signature 6:	4b67
Additional Hang Signature 7:	4b67857320e07e877cd0641d73a6b29a

```

Are you using the Microsoft Store version of WDD?: Yes

**Comment by Hexaae** [comment]
Crashed again updating to 5.3.0.0 automatically in background from MS Store:

```
Descrizione
A problem caused this program to stop interacting with Windows.
Percorso dell'applicazione che ha generato l'errore:	C:\Program Files\WindowsApps\38719TimothyJohnson.WinDynamicDesktop_5.2.1.0_x64__rfarkse4rn21a\WinDynamicDesktop\WinDynamicDesktop.exe

Firma del problema
Nome evento problema:	MoAppHang
Package Full Name:	38719TimothyJohnson.WinDynamicDesktop_5.2.1.0_x64__rfarkse4rn21a
Application Name:	praid:App
Application Version:	5.2.1.0
Application Timestamp:	63cf74d2
Hang Signature:	196a
Hang Type:	2097152
Versione SO:	10.0.22621.2.0.0.256.48
ID impostazioni locali:	1040
Additional Hang Signature 1:	196a76c64739cd1c43af4ec94a4e963a
Additional Hang Signature 2:	b72d
Additional Hang Signature 3:	b72d09349283ee689c1b665719f88007
Additional Hang Signature 4:	196a
Additional Hang Signature 5:	196a76c64739cd1c43af4ec94a4e963a
Additional Hang Signature 6:	b72d
Additional Hang Signature 7:	b72d09349283ee689c1b665719f88007

```

Please, find a clean way to kill running version and install new one while WDD is running.

**Comment by t1m0thyj** [comment]
This will be fixed in the next release, but the fix won't take effect until the release after next because it must be present in the version being updated _from_.

**Comment by t1m0thyj** [comment]
Fixed in WinDD 5.4.0
